quickstarts/hello-rules/README.md

```diff
@@ -0,0 +1,3 @@
+## Red Hat Decision Manager Kie Server Quickstart
+
+This quickstart is intend to be used with the [RHDM Kie Server](https://github.com/jboss-container-images/rhdm-7-openshift-image/tree/rhdm70-dev/kieserver) image.
```

Comment:
@spolti Can you add here steps how to configure and run hellorules-client?

Reply:
Sure, I have it done for another issue, just have to retest and update for rhdm image.